bin/travis/setupphp.sh

```diff
@@ -2,4 +2,4 @@
 
 add-apt-repository -y ppa:ondrej/php5
 apt-get update -q
-apt-get install -y php5-cli php5-common php5-gd php5-xsl php5-intl
+sudo apt-get install -q -y --force-yes php5-cli php5-common php5-gd php5-xsl php5-intl
```

Comment:
I did not use sudo in .travis.yml for this one, as I try to keep track if our sudo use for possibility to use travis container infrastructure. But I think they will fix that so if you want I can change it back to sudo in the yml file again.